packages/medusa/src/api/routes/admin/order-edits/index.ts

```diff
@@ -0,0 +1,28 @@
+import { Router } from "express"
+import middlewares from "../../../middlewares"
+import { isFeatureFlagEnabled } from "../../../middlewares/feature-flag-enabled"
+import OrderEditingFeatureFlag from "../../../../loaders/feature-flags/order-editing"
+import { DeleteResponse } from "../../../../types/common"
+import { FlagRouter } from "../../../../utils/flag-router"
+
+const route = Router()
+
+export default (app, featureFlagRouter: FlagRouter) => {
+  app.use(
+    "/order-edits",
+    isFeatureFlagEnabled(OrderEditingFeatureFlag.key),
+    route
+  )
+
+  route.delete(
+    "/:edit_id",
+    middlewares.wrap(require("./delete-order-edit").default)
+  )
+
+  return app
+}
+
+export type AdminOrderEditDeleteRes = DeleteResponse
```

Comment:
suggestion: What do you think of that?
```ts
export ( DeleteResponse as AdminOrderEditDeleteRes ) from "../../../../types/common"
```

also, I would suggest to move that to `types/order-edits` and instead of the above, to do something like

```ts
export ( AdminOrderEditDeleteRes ) from "../../../../types/order-edit"
```

Reply:
hmm, I don't love it, and it's not really something we do in other places so it might just cause a bit of confusion, but I don't care so much as to fight tooth and nail for it 😂 
@olivermrbl wdyt should we change it up?

Reply:
I don't have any strong opinions on it, but I'd suggest we follow the general convention, for now, to avoid introducing unnecessary clean-up tasks